api/controllers/UserController.js

```diff
@@ -569,62 +659,98 @@ module.exports = {
               countUnverified: 1,
             },
           }));
+          pendingLogs.push({
+            type: 'info',
+            message: `[UserController->update] User ${user._id.toString()} is becoming visible. Updated list counts`,
+          });
         }
         promises.push(user.save());
+        pendingLogs.push({
+          type: 'info',
+          message: `[UserController->update] User ${user._id.toString()} saved successfully`,
+        });
         if (!user.hidden) {
           promises.push(EmailService.sendAuthToProfile(user, request.auth.credentials));
+          pendingLogs.push({
+            type: 'info',
+            message: `[UserController->update] Sent auth_to_profile email to user ${user.email}`,
+          });
         }
       } else if (!user.hidden) {
         const notification = { type: 'admin_edit', user, createdBy: request.auth.credentials };
         promises.push(NotificationService.send(notification));
+        pendingLogs.push({
+          type: 'info',
+          message: `[UserController->update] Sent admin_edit notification to user ${user.id}`,
+        });
       }
     }
     promises.push(GSSSyncService.synchronizeUser(user));
-    promises.push(OutlookService.synchronizeUser(user));
-    try {
-      await Promise.all(promises);
-    } catch (err) {
-      return user;
+    pendingLogs.push({
+      type: 'info',
+      message: `[UserController->update] Synchronized user ${user.id} with google spreadsheet`,
+    });
+    // promises.push(OutlookService.synchronizeUser(user));
+    await Promise.all(promises);
+    for (let i = 0; i < pendingLogs.length; i += 1) {
+      logger.log(pendingLogs[i]);
     }
     return user;
   },
 
   async destroy(request, reply) {
     if (!request.auth.credentials.is_admin
       && request.auth.credentials._id.toString() !== request.params.id) {
+      logger.warn(
+        `[UserController->destroy] User ${request.auth.credentials._id.toString()} is not allowed to delete user ${request.params.id}`,
+      );
       return reply(Boom.forbidden('You are not allowed to delete this account'));
     }
 
-    logger.debug('[UserController] (destroy) model = user, query =', request.query, { request });
-
     const user = await User.findOne({ _id: request.params.id });
     if (!user) {
+      logger.warn(
+        `[UserController->destroy] Could not find user ${request.params.id}`,
+      );
       throw Boom.notFound();
     }
     await EmailService.sendAdminDelete(user, request.auth.credentials);
+    logger.info(
+      `[UserController->destroy] Removing user ${request.params.id}`,
+    );
```

Comment:
this one precedes the DB operation ;)

you were just testing to see if I read the whole PR, weren't you?

Reply:
Should be fixed in my last commit.